packages/@ember/string/index.ts

```diff
@@ -53,7 +53,7 @@ const UNDERSCORE_CACHE = new Cache<string, string>(1000, str =>
     .toLowerCase()
 );
 
-const STRING_CAPITALIZE_REGEXP = /(^|\/)([a-z\u00C0-\u024F])/g;
+const STRING_CAPITALIZE_REGEXP = /(^|(?<!\/)\/(?!\/))([a-z\u00C0-\u024F])/g;
```

Comment:
uses [positive and negative lookahead](https://www.regular-expressions.info/lookaround.html)